shared/src/test/factories.ts

```diff
@@ -221,7 +221,7 @@ export const generateProfileData = (
     responsibleOwnerName: `some-responsible-owner-name-${randomInt()}`,
     tradeName: `some-trade-name-${randomInt()}`,
     industryId: industry.id,
-    legalStructureId: randomLegalStructure().id,
+    legalStructureId: undefined,
```

Comment:
Why this change? A generator isn't necessarily representing "initial" state, it's representing a random state, so changing the onboarding doesn't seem like this should change?

Reply:
agree, will revert